src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,138 @@
+import { useEffect, useState } from 'react';
+
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+// eslint-disable-next-line import/extensions
+import { PeoplePageTitle } from '../page/PeoplePageTitle';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useSearchParams } from 'react-router-dom';
+import { QueryParams } from '../enum/queryParams.enum';
+import { Sex } from '../enum/sex.enum';
 
 export const PeoplePage = () => {
+  // #region states
+  const [loading, setLoading] = useState(true);
+  const [fetchPeopleError, setFetchPeopleError] = useState(false);
```

Comment:
Fetch??? This prefix is ​​only for fetch functions, NOT for booleans. Change it!